common/engine/keyboardprocessor/include/keyman/keyboardprocessor.h.in

```diff
@@ -490,14 +490,12 @@ enum km_kbp_action_type {
   KM_KBP_IT_ALERT       = 3,  // The keyboard has triggered a alert/beep/bell.
   KM_KBP_IT_BACK        = 4,  // Delete the codepoint preceding the insertion point.
   KM_KBP_IT_PERSIST_OPT = 5,  // The indicated option needs to be stored.
-  KM_KBP_IT_RESET_OPT   = 6,  // Set the indicated option from the last
```

Comment:
This is also being removed by other PRs but I assume that won't cause any trouble.